examples/react-apollo/package.json

```
@@ -3,21 +3,21 @@
   "version": "0.1.11",
   "private": true,
   "dependencies": {
-    "@apollo/client": "^3.6.9",
+    "@apollo/client": "^3.7.14",
     "@mantine/core": "^4.2.12",
     "@mantine/dropzone": "^4.2.12",
     "@mantine/hooks": "^4.2.12",
     "@mantine/notifications": "^4.2.12",
     "@mantine/prism": "^4.2.12",
-    "@nhost/react": "*",
-    "@nhost/react-apollo": "*",
-    "graphql": "15.7.2",
+    "@nhost/react": "^2.0.16",
```

```suggestion
    "@nhost/react": "*",
    "@nhost/react-apollo": "*",
```

This will always use the latest version that is available in the monorepo.